gulpfile.js/tasks/html.js

```diff
@@ -37,7 +37,7 @@ var htmlTask = function() {
     .on('error', handleErrors)
     .pipe(gulpif(global.production, htmlmin(config.tasks.html.htmlmin)))
     .pipe(gulp.dest(paths.dest))
-    .pipe(browserSync.stream())
+    .on('end', browserSync.reload)
```

Comment:
How does this compare with doing

```js
.pipe(browserSync.stream({once: true}));
```

as outlined in the [docs](https://www.browsersync.io/docs/api#api-stream)?